src/redux/features/taskBoardSlice.tsx

```diff
@@ -44,6 +44,15 @@ const taskBoardSlice = createSlice({
           : state.tasks
       state.filteredTasks = filteredTasks
     },
+    setFilteredAsignee: (state, action) => {
```

Comment:
let's type the `action` as well, please refer to RTK [Typescript doc](https://redux-toolkit.js.org/usage/usage-with-typescript)

@arpandhakal @aatbip @rrojan

Reply:
@saugatmaharjan Added types to `action`s in c8792f7e4af6ff1c7630e4e7e40ec55049bb6a34 commit

cc: @arpandhakal @rrojan